pytorch_lightning/tuner/lr_finder.py

```diff
@@ -115,8 +115,8 @@ def func():
                 optimizers = [optim_conf["optimizer"]]
             elif isinstance(optim_conf, (list, tuple)) and isinstance(optim_conf[0], dict):
                 optimizers = [opt_dict["optimizer"] for opt_dict in optim_conf]
-            elif isinstance(optim_conf, (list, tuple)):
-                optimizers = [optim_conf]
+            elif isinstance(optim_conf, (list, tuple)) and all(isinstance(opt, Optimizer) for opt in optim_conf):
```

Comment:
What a coincidence. Today I ran into the same issue and observed exactly this problem here. I was in the middle of converting this file to BoringModel usage. Good that I saw this now!

Great work!